config.example/helm/binderhub-config.yml

```diff
@@ -15,3 +15,6 @@ jupyterhub:
     enabled: true
     timeout: 3600 # Delete Pods that have been inactive for over n seconds
     every: 600 # Check for inactive Pods ever n seconds
+
+dind:
+  enabled: true
```

Comment:
This config is pretty opaque unless you know about Docker in Docker. Can you add a comment explaining what this config and why you'd want it enabled or disabled? Either here or in the doc.

(I'd appreciate the clarification myself! I may have missed it, but a quick browse of the BinderHub docs doesn't explain why you'd want to do this...)